pkg/datapath/types/node.go

```diff
@@ -75,6 +75,9 @@ type LocalNodeConfiguration struct {
 	// EnableIPSec enables IPSec routes
 	EnableIPSec bool
 
+	// EnableIPSecEncryptedOverlay enables IPSec routes for overlay traffic
+	EnableIPSecEncryptedOverlay bool
```

Comment:
Does this require `EnableIPSec`? If so, please update the godocs.

Reply:
Added in the docs of the feature flag - latest commit.